neural_compressor/torch/algorithms/weight_only/awq.py

```diff
@@ -429,14 +429,15 @@ def apply_quantize_with_clip(self, return_int=False):
         """
         # apply quantization and clip
         logger.info("Quantizing the AWQ optimized fp32 model")
-        from .rtn import rtn_quantize
+        from .rtn import RTNQuantizer
 
-        self.model = rtn_quantize(
+        rtn_quantizer = RTNQuantizer()
+        rtn_quantizer.config = self.weight_config
+        rtn_quantizer.quantize(
             self.model,
-            num_bits=self.bits,
+            bits=self.bits,
             group_size=self.group_size,
             scheme=self.scheme,
-            weight_config=self.weight_config,
```

Comment:
This is not correct, AWQ is using weight_config to notice RTN how to quantize with clip range.

Reply:
weight_config here is set as a member variables of RTNQuantizer. You can see it on Line-435 `rtn_quantizer.config = self.weight_config`